tools/create_index/index_generator.js

```diff
@@ -0,0 +1,50 @@
+var fs = require('fs')
+
+// We will fill in this object with our data
```

Comment:
This comment isn't so useful, I'd either drop it, or add one that describes the structure of what the object will be filled with.

Reply:
Improved it 👍